src/test/fixtures/archive/contents/testbook1-shortid

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Are these manually updated? If so, would it make sense to use something like https://github.com/vcr/vcr to record the fixtures?

Or drop in a `fetch` implementation that records/plays-back the fixtures? like https://www.npmjs.com/package/fetch-vcr

Reply:
Ok, if it is fake data then could it be lorem-ipsum-style ... whatever is the minimum used by the tests? That would help make it clear whether a particular combination of elements is being tested rather than the content just being in the original copy/paste

Reply:
is that not what it is?

Reply:
Do we use (or plan to use) fields like `submitlog`, `legacy_id`, `stateid`, `parentId` (bingo!, got all 3 ways to combine word+id)?

and others like `parentVersion`, `legacy_version`, `history`, `buyLink`, `doctype`, `licensors`, `publishers`, `authors`?